packages/@okta/i18n/src/properties/login_ok_PL.properties

```diff
@@ -940,6 +940,8 @@ oie.user.profile.division = 》Ðîṽîšîöñ 䀕ヸ€홝한Ӝฐโ《
 oie.user.profile.department = 》Ðéþåŕţɱéñţ ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.user.profile.managerId = 》Ṁåñåĝéŕ ÎÐ ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.user.profile.manager = 》Ṁåñåĝéŕ ヸ€홝한Ӝฐโ《
+oie.selfservice.unlock_user.landing.to.app.success.message = 》Åççöûñţ šûççéššƒûļļý ûñļöçķéð¡ Ṽéŕîƒý ýöûŕ åççöûñţ ŵîţĥ å šéçûŕîţý ɱéţĥöð ţö çöñţîñûé· ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
+oie.selfservice.unlock_user.landing.to.app.signing.in.message = 》Åççöûñţ šûççéššƒûļļý ûñļöçķéð¡ 䀕ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
```

Comment:
Already exists in master see: https://github.com/okta/okta-signin-widget/blob/b00690bacaef60dec1e1508c04993ad1dcea56ec/packages/@okta/i18n/src/properties/login_ok_PL.properties#L962-L963